.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "import/no-extraneous-dependencies": "off"
```

Comment:
We have some little changes in `.eslintrc.json`,  check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/react-redux), according to the project requirement we need to set the linter's correctly with the standard Microverse config file without changing it.

Reply:
it another file from previous project I added accidentally, 
thanks for the note.
Done :+1: